core-bundle/tests/Contao/ControllerTest.php

```diff
@@ -90,8 +90,13 @@ public function testReturnsTheTimeZones(): void
         $this->assertCount(13, $timeZones['United States']);
     }
 
+    /**
+     * @group legacy
+     */
     public function testGeneratesTheMargin(): void
     {
+        $this->expectDeprecation('%sUsing Contao\Controller::generateMargin is deprecated%s');
```

Comment:
```suggestion
        $this->expectDeprecation('Since contao/core-bundle 4.13: Using Contao\Controller::generateMargin is deprecated%s');
```
```suggestion
        $this->expectDeprecation('%sUsing Contao\Controller::generateMargin is deprecated%s');
```